tests/lib/Controller/ConfigControllerTest.php

```diff
@@ -1356,4 +1357,77 @@ public function testSetupIntegrationProjectFoldersSetUp():void {
 		$this->assertArrayHasKey('oPUserAppPassword', $data);
 		$this->assertEquals("gliAcIJ3RwcgpF6ijPramBVzujfSQwJw2AVcz3Uj7bdXqxDbmkSukQhljAUf9HXItQTglvfx", $data['oPUserAppPassword']);
 	}
+
+	public function testsignTOSForUserOpenProject():void {
```

Comment:
cammelCase here also

```suggestion
	public function testSignTOSForUserOpenProject():void {
```